.circleci/config.yml

```diff
@@ -503,9 +503,6 @@ workflows:
       - run-e2e-test-IOS:
           requires:
             - compile-typescript
-          filters:
-            branches:
-              only: master
```

Comment:
Could you restore this part?

Reply:
Thanks, I thought I already pushed this: https://github.com/pagopa/io-app/pull/3986/commits/3dfd040cde9426ab5bbc1a7ee1578504aa208b37 :)